lib/criteo_api.js

```diff
@@ -17,7 +15,7 @@ const sha512 = require('js-sha512').sha512;
  */
 class Criteo_API_Client extends API_Client {
 
-    constructor(id, secret, timeout, host = 'api.criteo.com', endpoint = '', version = '2023-10') {
+    constructor(id, secret, timeout, host = 'api.criteo.com', endpoint = '', version = '2024-07') {
```

Comment:
/nit not a big fan of having the version being passed as a string in the constructor. id rather have it declared as a constant. but really its what youd prefer

Reply:
@varun27896 yea its interesting, will probably leave it since this is logic from the forked Repo. I might create a ticket later to see if we could just add an api call to the dwh-criteo-datasource repo instead of using this repository

Reply:
We did this originally to avoid the necessity for PRs like this one. The implementer of this library can decide what version they want to use.